python/ccxt/base/exchange.py

```diff
@@ -1093,31 +1102,12 @@ def load_accounts(self, reload=False, params={}):
         self.accountsById = self.index_by(self.accounts, 'id')
         return self.accounts
 
-    def populate_fees(self):
-        if not (hasattr(self, 'markets') or hasattr(self, 'currencies')):
-            return
-
-        for currency, data in self.currencies.items():  # try load withdrawal fees from currencies
-            if 'fee' in data and data['fee'] is not None:
-                self.fees['funding']['withdraw'][currency] = data['fee']
-                self.fees['funding']['fee_loaded'] = True
-
-        # find a way to populate trading fees from markets
-
-    def load_fees(self):
+    def load_fees(self, reload=False):
         self.load_markets()
-        self.populate_fees()
-        if not (self.has['fetchTradingFees'] or self.has['fetchFundingFees']):
-            return self.fees
-
-        fetched_fees = self.fetch_fees()
-        if fetched_fees['funding']:
-            self.fees['funding']['fee_loaded'] = True
-        if fetched_fees['trading']:
-            self.fees['trading']['fee_loaded'] = True
-
-        self.fees = self.deep_extend(self.fees, fetched_fees)
-        return self.fees
+        if not reload:
```

Comment:
This should probably be `if reload` ) no? )

Reply:
its working on my end?

Reply:
It "seems to work", but it will send out a `fetch_fees` http request even if `reload=False`.

Reply:
https://github.com/ccxt/ccxt/pull/4948/files#diff-a288301ab3ea0b68559e658361797d63R197